packages/foam-vscode/static/dataviz/graph.js

```diff
@@ -223,12 +243,35 @@ function initDataviz(channel) {
     .onBackgroundClick(event => {
       Actions.selectNode(null, event.getModifierState('Shift'));
     });
+    const gui = new dat.gui.GUI();
+    gui.add(model, 'view', {Default: "default", Tags: "tags", All: "all"})
+      .name('View')
+      .onFinishChange(function(){
+        Actions.evaluate();
+    });
 }
 
 function augmentGraphInfo(data) {
   Object.values(data.nodes).forEach(node => {
     node.neighbors = [];
     node.links = [];
+    if(node.tags && node.tags.length > 0){
+      node.tags.forEach(tag => {
+        const tagNode = {
+          id: tag.label,
+          title: tag.label,
+          type: 'tag',
+          properties: {color: d3.interpolateTurbo(Math.random())},
```

Comment:
this should follow the stying pattern of all other nodes. In fact, I believe it's supported out of the box because we support styling nodes by type. see https://github.com/foambubble/foam/blob/master/docs/features/graph-visualisation.md

Reply:
It helps me to differentiate between tags to have them multicolored, but a uniform style is probably the more conventional approach. I originally used that approach, but then I found myself searching for the same tag over and over again. I have a much better approach to tag filtering sitting in a branch though, so once we have these initial changes worked out, I will improve the tag filtering.

Reply:
that sounds good. I am also open to improvements around the styling of the graph, I just prefer a consistent approach, so that:
1. there is one way of going things for users
2. when there are updates, they can pushed across the domain

So yeah, let's start with our standard styling approach, and in another PR I am happy to consider changes to the approach